src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdaterTest.java

```diff
@@ -99,6 +100,128 @@ public void should_update_case_data_with_exception_record_if_no_warnings() {
         );
     }
 
+    @Test
+    public void should_update_case_data_with_exception_record_if_null_case_update_details_no_warnings() {
```

Comment:
something wrong with that sentence, I don't understand

Reply:
renamed to `should_update_case_with_exception_record_with_no_warnings_if_null_case_update_details_no_warnings`